tests/unit/elements/Button.spec.js

```diff
@@ -0,0 +1,90 @@
+import { shallowMount } from '@vue/test-utils'
+
+import Button from '@/elements/Button/Button'
+
+describe('Button.vue', () => {
+    const ROUNDNESS = 'complete'
```

Comment:
Order of constants should be the in the order of tests
The order of tests is explained below

Reply:
Solved.